downlink/python/anvil/__init__.py

```diff
@@ -66,7 +66,9 @@ def get_name(self):
 class BlobMedia(Media):
     def __init__(self, content_type, content, name=None):
         self._content_type = content_type
-        if isinstance(content, _unicode_string_type):
+        if content is None:
```

Comment:
This check is very specific - it only catches `None`s. Could we try checking for "anything that isn't a byte-string"? (We could have a counterpart to `_unicode_string_type`, which is set to `str` in Python 2 and `bytes` in Python 3, then produce this error `if not isinstance (content, _byte_string_type):`

Reply:
Anvil basically writes itself!